app/assets/javascripts/admin.js

```diff
@@ -1,4 +1,4 @@
 //= require rails-ujs
 //= require jquery3
 //= require popper
-//= require bootstrap-sprockets
```

Comment:
@Mons-web We should always prefer to use Sprockets-mastered script since Rails uses Sprockets :) Also this is the recommendation of the gem itself https://github.com/twbs/bootstrap-rubygem